persistence/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/PersistenceComponents.scala

```diff
@@ -31,9 +31,11 @@ trait ReadSidePersistenceComponents extends WriteSidePersistenceComponents {
   def actorSystem: ActorSystem
   def executionContext: ExecutionContext
   def materializer: Materializer
+
+  @deprecated(message = "prefer `config` using typesafe Config instead", since = "1.4.0")
```

Comment:
I don't think this deprecation warning belongs here now, as there is no `config` member.